src/Microsoft.Extensions.Configuration.AzureAppConfiguration/OfflineFileCache.cs

```diff
@@ -130,12 +93,8 @@ public string Import(AzureAppConfigurationOptions options)
                                     data = reader.ReadAsString();

```

Comment:
I'm wondering if we should add a version property representing the schema version. This is one way we can ensure that we don't interact with an old offline cache file.

Reply:
If we do read an old offline cache file, we won't be able to decrypt it with the new DataProtector anyway. So `Import` is definitely going to fail. Do you have any scenario in mind where we might need to be more careful?

Reply:
That's fine, if we need versioning because we change some formatting then we can worry about that in the future and consider v1 to be the object that doesn't have version.

Reply:
What's going to be the expectation for customers who reference the new provider, and configure the offline file cache to point to the same file path?

Reply:
In the cache-first approach during startup, I think we should swallow (and log in future) any exceptions while reading from cache file and continue loading from server.
The first time data is loaded from server, their cache file will be updated with new data encrypted using data protector.
If they also get a server exception on the first try loading from server, then we can throw the server exception.